gtsfm/two_view_estimator.py

```diff
@@ -136,8 +139,19 @@ def create_computation_graph(
             num_inliers_gt_model, inlier_ratio_gt_model = None, None
             v_corr_idxs_inlier_mask_gt = None
 
+        # Note: homography estimation threshold must match the E / F thresholds for #inliers to be comparable
+        # TODO(johnwlambert): H_graph is currently unused, but in a future PR pose from homography will be extracted.
+        H_graph, H_inlier_idxs, inlier_ratio_H, num_inliers_H = dask.delayed(self._homography_verifier.verify, nout=4)(
+            keypoints_i1_graph,
+            keypoints_i2_graph,
+            match_indices=corr_idxs_graph,
+            estimation_threshold_px=self._verifier._estimation_threshold_px,
+        )
+
```

Comment:
we should create two different functions in the `TwoViewEstimator`: one for verification (E/F/H) and the other for reporting/metrics. We have way too much logic in one function.

Reply:
There is a lot here. This might be better for a future PR since it will involve a lot of refactoring. Can you sketch out briefly in a comment or pseudocode the way you envision us structuring things here?